File: challenge-330/hvukman/lua/330_p1.lua
function Nice (a)
    local not_good = false
    local new
    for i = 1, string.len(a) do
        local char = string.sub(a, i, i)
        if string.match(char, "%d") then
            new= string.sub(a, 1, i-2) .. string.sub(a, i+1, string.len(a))
            not_good= true
            break
        end

    end
    if not_good==true then
        return Nice(new)
    end
    print(a)
    return a
end

Nice("cab12")
Nice("xy99")
Nice("pa1erl")

File: challenge-330/hvukman/F#/330_p1.fs
let str = "cab12"
let pat = "[A-Za-z]\d"
[<TailCall>]
let rec rec_repl n = if Regex.Match(n,pat).Success=true then rec_repl (Regex.Replace(n,pat,"")) else printf "%s\n" n 
rec_repl "cab12"
rec_repl "xy99"
rec_repl "pa1erl"
